torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -479,6 +479,24 @@ const std::vector<std::string> functions = {
                 return grad_output._grad_sum_to_size(self_size), grad_tensor1, grad_tensor2, None
             return result, backward
 
+        def _autocast_to_full_precision(self, cuda_enabled : bool, cpu_enabled : bool):
+            self_dtype = self.dtype
```

Comment:
i wonder if the self.dtype forces materialization of `self` and prevents fusion... something to look into

Reply:
I remember seeing self.dtype parsed to a `prim::dtype` op. :crossed_fingers: